druid/src/theme.rs

```diff
@@ -186,5 +186,5 @@ pub(crate) fn add_to_env(env: Env) -> Env {
 
 #[deprecated(since = "0.7.0", note = "use Env::default() instead")]
```

Comment:
I think we should also change the note to following
```
#[deprecated(since = "0.7.0", note = "use Env::with_default_i10n() instead")]
```

or `Env::empty()`. You know best :)

Reply:
Can we just remove this? I don't think this is a popular function :thinking:

Reply:
agree, there's no good way to use this right now.